.github/workflows/build-container.yaml

```
@@ -0,0 +1,65 @@
+name: Building and publishing container
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*.*.*"
+      - "force-docker-build"
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=openwpm/openwpm
+          TAGS="${DOCKER_IMAGE}:${GITHUB_SHA}"
+          # If it's a tagged build it should get published as such
+          if [[ $GITHUB_REF == refs/tags/v* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/v}
+            TAGS="$TAGS,${DOCKER_IMAGE}:$VERSION"
+          fi
+          # If it is on master it should get tagged as latest
+          if [[ $GITHUB_REF == refs/heads/master ]]; then
+              TAGS="$TAGS,${DOCKER_IMAGE}:latest"
+          fi
+          echo ::set-output name=tags::${TAGS}
+
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      -       
+        name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      -     
+        name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - 
```

```suggestion
      - name: Login to Docker Hub
```
We could add a name to the login action.
And perhaps we could have the hyphens on the same line as  `name:` throughout the file? That's an insignificant detail really, I just found it a little confusing.